src/android/Mapbox.java

```diff
@@ -168,7 +183,23 @@ public void run() {
             FrameLayout.LayoutParams params = new FrameLayout.LayoutParams(webViewWidth - left - right, webViewHeight - top - bottom);
             params.setMargins(left, top, right, bottom);
             mapView.setLayoutParams(params);
-
+			//@anothar we have to override back button or app will just close
+            mapView.setOnKeyListener(new View.OnKeyListener() {
```

Comment:
This google maps plugin https://github.com/mapsplugin/cordova-plugin-googlemaps does seem to handle that differently, doesn't it? I see no key handling in there.